core/station/api/src/resource.rs

```diff
@@ -7,6 +7,7 @@ pub enum ResourceDTO {
     Account(AccountResourceActionDTO),
     AddressBook(ResourceActionDTO),
     ChangeCanister(ChangeCanisterResourceActionDTO),
+    SetDisasterRecovery(ChangeCanisterResourceActionDTO),
```

Comment:
Any reason not to add this under the System `Resource` and just extend the variant of `SystemResourceActionDTO` with something like `ManageDisasterRecovery`?